pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.usort]
+
+[tool.usort.known]
+
+first_party = ["torch", "torchvision"]
```

Comment:
If I understand the "first-party" feature correctly, this means the `torch` imports will be done last, right?

For torchvision it's not an issue but for torch we actually want it to be the first import, because `import torch` does a bunch of stuff in the background like resolving dynamic links and other magic stuff

Reply:
> If I understand the "first-party" feature correctly, this means the torch imports will be done last, right?

What this does is move `torch` into the first party category. By default only local packages, e.g. `torchvision`, go into that block. Have a look at the [sorting documentation](https://usort.readthedocs.io/en/latest/guide.html#sorting) for an example of the blocks.

> `import torch` does a bunch of stuff in the background like resolving dynamic links and other magic stuff

Does that mean we need to always import `torch` before other PyTorch derivatives like `torchvision` or does it even need to be imported before all other third party packages?